components/Header.js

```diff
@@ -1,5 +1,17 @@
+import styled from "styled-components";
+
 export default function Header() {
     return (
-        <h1>Minigame Collection</h1>
+        <>
+            <Head>
+                <h1>Minigame Collection</h1>
+            </Head>
+        </>
     )
 }
+
+export const Head = styled.header`
```

Comment:
you should choose a different name. NextJS provides us with a component with the same name. This might lead to confusion. More info about [this](https://nextjs.org/docs/api-reference/next/head).